index.js

```diff
@@ -324,10 +324,14 @@ function showNotification(notificationObj) {
           })
         }
 
+
+        // Display time per notification basis.
+        var displayTime = (notificationObj.displayTime ? notificationObj.displayTime : config.displayTime);
```

Comment:
Not sure how that can be mistaken for a function. I read up on the syntax of the grouping operating and it doesn't matter, the braces are totally optional ^^. It just seemed funny to me coming from a PHP background an being used to `(condition) ? true : false`